fuel/src/main/kotlin/com/github/kittinunf/fuel/core/Request.kt

```diff
@@ -152,8 +152,8 @@ class Request(
 
     fun authenticate(username: String, password: String): Request {
         val auth = "$username:$password"
-        val encodedAuth = Base64.encode(auth.toByteArray(), Base64.NO_WRAP)
-        return header("Authorization" to "Basic " + String(encodedAuth))
+        val encodedAuth = auth.toByteArray().encodeBase64()
+        return header("Authorization" to "Basic $encodedAuth")
```

Comment:
I assume this auto converts into a string? May want to do it explicitly still so new people see that encodedAuth is *not* a String?